### DISCUSSION TITLE
error with loading the gamma-processed data

### DISCUSSION THREAD

**Discussion by keepfighting666** [insarlab/MintPy#1260] [Q&A]
When I load the gamma-processed data, the following error occurs. I checked the previous issues similar to mine but couldn't find a reasonable solution. To add more context, although I didn't organize my files according to the example structure, I provided the correct file paths in the configuration file. What can I do to solve it?
![image](https://github.com/user-attachments/assets/c2c031ea-ec5a-458c-89e7-ad579d11c32e)
![image](https://github.com/user-attachments/assets/3e080678-546c-4753-9a49-41fa8d398c8b)

**Comment by welcome** [comment]
👋  Thanks for opening your first issue here! Please filled out the template with as much details as possible. We appreciate that you took the time to contribute!
Make sure you read our [contributing guidelines](https://github.com/insarlab/MintPy/blob/master/docs/CONTRIBUTING.md).

**Comment by yunjunz** [comment]
please use the recommended file structure, which mintpy relies on to grab all the metadata, gamma's file structure can be very different from one to another.

**Reply by keepfighting666** [reply]
Thanks a lot.
